internal/resources/azure/util.go

```diff
@@ -0,0 +1,20 @@
+package azure
+
+import (
+	"github.com/shopspring/decimal"
+)
+
+func strPtr(s string) *string {
```

Comment:
Off topic for this PR, but since we use these everywhere what are your thoughts about moving these to their own package longer term? I've been hesitant because of the general golang advice of avoiding `util` packages.

Reply:
I think it okay sometimes to bend a rule/good practice in favor of simplicity/usability when it makes sense. But I'm not very familiar with Go way to make such suggestion yet :D